optuna/notebooks/jupytercon_demo/RAPIDS_xfeat_Optuna-GPU.ipynb

```diff
@@ -0,0 +1,1803 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Image\n",
+    "\n",
+    "f = optuna.visualization.plot_param_importances(study)\n",
+    "Image(f.to_image(format=\"png\", engine='kaleido'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following is a slice plot to better understand the parameter relationships. We see how the change in the parameter affects the performance of the model."
```

Comment:
Maybe a little more detailed commentary? "We can see that choosing the correct penalty and number of features is critical to model accuracy. Some parameters like l1_ratio had very little impact. Rather than spending our time tuning these manually, Optuna HPO has handled it for us."